app/app/static/custom/ourscript.js

```diff
@@ -342,3 +342,43 @@ window.onload = async (event) => {
 	rerenderTables();
 	
 };
+
+$('#dateSubmit').on('click', function(e) {
+	const startDate = document.getElementById("startDate").value;
+	const endDate = document.getElementById("endDate").value;
+	let startTime = startDate + " 00:00:00.0"
+	let endTime = endDate + " 23:59:59.9"
```

Comment:
Use JavaScript string template

Reply:
Also this can just use `const` instead of `let`

Reply:
As i said before, i'm not too confident in my js, so its always great to get feedback about correct syntax

Reply:
Done in next commit